### ISSUE TITLE
Select the ledger derivation path before getting the addresses

### ISSUE THREAD

**Issue by clementbalestrat** [Synthetixio/synthetix-mintr#92]

**Comment by bachstatter** [comment]
I would think  defaulting is fine and then everytime they pick we save it to localstorage?

Up do you, I can obv make it default to unselected

**Comment by clementbalestrat** [comment]
@bachstatter Yeah I guess it's fine loading the wrong derivation path for the first time.

**Comment by clementbalestrat** [comment]
@bachstatter though I just found out if I refresh and "ledger live" is selected by default, it will load the wrong derivation path.

**Comment by clementbalestrat** [comment]
Maybe we could write some tests to make sure we init Trezor and Ledger with the correct params, regarding the derivation path, what is save in the cache...etc. What do you think?
